.github/cicd/functions.sh

```diff
@@ -0,0 +1,174 @@
+export BRANCH_LOWER=`echo "${BRANCH_NAME}" | awk '{print tolower($0)}'`
+function variablePopulation() {
+    if [ "${BRANCH_LOWER}" == "develop" ] || [ "${BRANCH_LOWER}" == "master" ];
+    then
+        export SUFFIX=""
+        export CHANGE_BRANCH="$BRANCH_NAME"
+    else
+        export SUFFIX="-${BRANCH_LOWER}";
+    fi
+    if [ "${APP_NAME}" == "test" ] || [ "${APP_NAME}" == "prod" ];
+    then
+        export DOTNET_PHASE="Release"
+    else
+        export DOTNET_PHASE="Development"
+    fi
+}
+
+variablePopulation
+
+function pipeline_args() {
+    export PIPELINE_ARGS="$*"
+}
+
+function build() {
+    source configurations/$2.conf
+    echo "Building $2 (${APP_NAME}) to $PROJECT_PREFIX-$3..."
+    buildPresent=$(oc get bc/"$APP_NAME-$BRANCH_LOWER" --ignore-not-found=true)
+    if [ -z "${buildPresent}" ];
```

Comment:
I might hit a few of these functions with some whitespace for readability later if that's ok

Reply:
Careful with whitespace - some places don't function right without the right amount.